pages/sunos/snoop.md

```diff
@@ -0,0 +1,24 @@
+# snoop
+
+> Network packeet sniffer.
+> SunOS equivalent of tcpdump.
+
+- Capture packets on network interface e1000g0:
+
+`snoop -d e1000g0`
+
+- Capture packets to a file:
+
+`snoop -o {{filename}}`
+
+- Display verbose protocol layer summary of packets from a file:
+
+`snoop -V -i {{filename}}`
+
+- Capture network packets that come from hostname and go to port 8000:
+
+`snoop to port 8000 from host {{hostname}}`
+
+- Capture and show network packets in hex and ASCII between two IP addresses:
```

Comment:
```suggestion
- Capture and show an hex-dump of network packets exchanged between two IP addresses:
```
Clearer.

Reply:
British pronunciation would use `an hex-dump`, while American English would use `a hex-dump`.

Reply:
We can go with the American pronunciation :+1: thanks for for the note.